documentation/src/main/markdown/packagemgrs/conan.md

```diff
@@ -6,28 +6,29 @@
 
 ## Overview
 
-[solution_name] has two detectors for Conan:
+[solution_name] has three detectors for Conan:
 
+* Conan 2 CLI detector
```

Comment:
@andrian-synopsys Do we really need to mention Conan 2 CLI and Conan 1 CLI? From the user context, isn't that irrelevant? That is internal/technical to Detect implementation. From user perspective the context is just Build and Buildless. We do support multiple versions of same package manager already (for example npm) From user perspective, identification of components for Conan v1 and v2 and cascading is all that matters

Reply:
Good question. My main goal was to describe how cascading differs depending on the version of Conan in use.
I'll see if I can accomplish this without getting into unnecessary details.

Reply:
Updated to not mention the two versions of CLI detectors. Conan versions are still mentioned in several places as functionality and requirements differ depending on the conan version.